Exercicio de casa feito

amrodrigues opened the pull request:
Resoluções de exercicios de casa e de aula.

amrodrigues committed:
Exercicio de casa feito

Resoluções de exercicios de casa e de aula.

amrodrigues committed:
Atualização com a resolução de Troco

davinyleticia commented:
Muito bom, gostei, fiz também usando class, mas poderia usar o objeto direto, está dica serve para mim também, como poderia ficar:

```
const bank = {
    saldo: 0,
    limite: 0,
    nomeTitular: '',
    consultaSaldo: function () {
        return this.saldo;
    },
    saque: function (valor) {
        if (this.saldo - valor < 0) {
            const saldoNegativo = this.saldo - valor;
            return saldoNegativo;
        }
        return this.saldo -= valor;
    },
    deposito: function (valor) {
        return this.saldo += valor;
    }
}
```

não precisa de instanciar a class

parabéns: approved  ✅

amrodrigues committed:
Atualização do troco

Usando o parseFloat e o toFixed para fixar o valor em duas casas decimais e acresecentando mais testes

amrodrigues committed:
FizzBuzz

Resolução do FizzBuzz e colocado os arquivos na pasta para aula dentro da pasta com meu nome

amrodrigues committed:
Atualização Conta Bancaria

Verfiquei que não tinha feito o teste corretamente do saque negativo e alteracao do novo limite

amrodrigues committed:
sincornização da pasta

amrodrigues committed:
Merge pull request #1 from reprograma/main

atualização do repo da reprograma

amrodrigues committed:
Projeto Final
